packages/metal-dom/src/domNamed.js

```diff
@@ -488,12 +511,20 @@ export function parent(element, selector) {
 
 /**
  * Registers a custom event.
- * @param {string} eventName The name of the custom event.
+ * @param {!string|!Object} eventName The name of the custom event or a regular
+ * expression object.
  * @param {!Object} customConfig An object with information about how the event
```

Comment:
This can now also be a function, so the param type and description should be updated accordingly.